cmd/bosun/w.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
 while echo "(RE)STARTING BOSUN"; do
-	go run main.go -w -dev -q  || exit
+	go run main.go -w -dev  || exit
```

Comment:
I don't generally use this script, but is this a change you actually mean to commit?